Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/common.js

```diff
@@ -197,4 +197,22 @@ function elapsedTime(startTimestamp, endTimestamp)
     return result;
 }
 
-export {deepCompare, ErrorDisplay, queryToParams, paramsToQuery, QueryModifier, escapeHTML, linkify, percentage, elapsedTime};
+function toGithubLink(file, suite) {
+    let baseURL = 'https://github.com/WebKit/WebKit/blob/main/';
```

Comment:
Hard-coded URLs like this are generally a red-flag in `Tools/Scripts/libraries`, we try to keep hard-coded WebKit specific foo in configuration or webkitcorepy. I'm not entirely certain that what I'm about to suggest is the best approach, but we have something similar with the concept of "archive_routes" in the `ViewRoutes` class where we define a specific behavior (in that case, linking to the diff of the test result) unique to layout-tests. We definitely need something like that additional layer of indirection here, because there are more suites than just the ones you've listed here.